packages/paste-website/src/component-examples/ListboxPrimitiveExamples.ts

```diff
@@ -0,0 +1,250 @@
+export const defaultExample = `
+const DefaultExample = () => {
+  const [items] = React.useState(['Item 1', 'Item 2', 'Item 3']);
+  const [selected, setSelected] = React.useState();
+  const listbox = useListboxPrimitiveState();
+  return (
+    <ListboxPrimitive {...listbox} aria-label="My listbox">
+      <Stack orientation="vertical" spacing="space0">
+        {items.map((item) => (
+          <ListboxPrimitiveItem
```

Comment:
Suggestion: could you add some minimal styles for selected options in the unstyled examples? I think it would help make it clearer that something is happening when you click an option.